pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
```

Comment:
il `--cov-report term-missing` serve per stampare la coverage quando vengono fatti i check automatici su github, non avendo ancora messo in piedi Coveralls. Ad esempio adesso non mi sembra sia possibile verificare qui la coverage dei test che hai aggiunto.
Se vuoi html basta lanciare `coverage html` appena finiti i test (puoi aggiungerlo al comando `test` in Makefile)

Reply:
Ri-aggiunto il `--cov term-missing`